payloadprocessor/to_hex.js

```diff
@@ -0,0 +1,16 @@
+ /**
+ * Converts a string payload to hex.
+ * 
+ * Created to add functinality found in Burp to solve Natas19
+ * https://www.youtube.com/watch?v=z3RtpWZ_R3Q
+ *
+ */
+
+function toHex(payload) {
```

Comment:
It needs to keep the previous name to work properly.

Reply:
ok